cli/src/main.rs

```diff
@@ -67,6 +67,7 @@ pub async fn main() {
     .subcommand(SubCommand::with_name("new")
       .about("Deploys your repository to a new app with one of the deploy configs from anycloud.json")
       .arg_from_usage("<DEPLOY_NAME> 'Specifies the name of the deploy config to use'")
+      .arg_from_usage("--app-id=[APP_ID] '[Optional] Specifies the application identifier'")
```

Comment:
nit remove Optional and add shorthand `-i, --app-id=[APP_ID] 'Specifies an optional application identifier'`